packages/commonwealth/client/scripts/views/components/header/login_selector.tsx

```diff
@@ -197,6 +199,22 @@ export const LoginSelectorMenuRight = ({
     localStorage.getItem('dark-mode-state') === 'on'
   );
 
+  const resetWalletConnectSession = async () => {
+    /**
+     * Imp to reset wc session on logout as subsequent login attempts fail
+     */
+    const chainbase = app.chain?.meta?.base;
+    const wallets = app.wallets.availableWallets(chainbase);
+
+    const wallet = wallets.find(
+      (w) =>
+        w instanceof WalletConnectWebWalletController ||
+        w instanceof TerraWalletConnectWebWalletController
```

Comment:
Possible bug here with terra. The terra walletconnect controller doesn't gracefully handle non-enabled wallets. Try logging in with something and then logging out on the terra page. You might see an error.

Reply:
Fix pushed.